profiles/common/modules/custom/nexteuropa_cookie_consent_kit/README.md

```diff
@@ -11,33 +11,23 @@ INSTALLATION
 
 CONFIGURATION
 -------------
-* Go to the _admin/config/system/nexteuropa-cookie-consent-kit_ page to enable or
-to disable the banner.
-* If it is required, when a site creates a persistent cookie, this cookie should
-be listed in your site's cookie_config.js.<br /> For more information, check
-<a href="https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Cookie+Consent+Kit+-+Technical+details">here</a>.<br />
-See also the DEVELOPER section in order to know how to inject the "cookie_config.js" file.
+* Go to the _admin/config/services/webtools_ page to enable/disable and configure the banner.
 * In case you would like to have the "Cookie Consent Kit" for 3rd party videos
-providers (YouTube, Vimeo and Dailymotion), you also
-need to enable the cookie videos.
+providers (YouTube, Vimeo and Dailymotion), you also need to enable the cookie 
+videos.
```

Comment:
This is a bit confusing: If I understand correctly it's the banner they need to enable? So perhaps this:
```suggestion
providers (YouTube, Vimeo and Dailymotion), you also need to enable the video 
cookie banner.
```